packages/core-webhooks/lib/manager.js

```diff
@@ -50,23 +50,24 @@ class WebhookManager {
     const matches = []
 
     for (const webhook of webhooks) {
-      if (!webhook.conditions) {
-        matches.push(webhook)
+      if (webhook.enabled == 1) {
```

Comment:
This should be one of the following.

`if (webhook.enabled)`
`if (webhook.enabled ===true)`
`if (!!webhook.enabled)`

Reply:
enabled is stored as 1 or 0 in sqlite (as tinyInt and not boolean). Enabled = 1, Disabled = 0